app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, old_name, new_name = command.split()
+    if old_name != new_name and cp == "cp" and len(cp) != 0:
+        with open(old_name, "r") as file_in:
+            with open(new_name, "w") as file_out:
```

Comment:
```
        with (open(old_name, "r") as file_in,
                 open(new_name, "w") as file_out):
```

Reply:
What do you mean?

Reply:
I think Bohdan had this in mind)))
```
with (
    open(old_name, "r") as file_in,
    open(new_name, "w") as file_out
):
```

Reply:
when I make this way I have invalid syntax error. If I use slash everything ok. That's why I changed with 2 WITH OPEN